devito/passes/clusters/aliases.py

```diff
@@ -1169,8 +1169,12 @@ def _pivot_min_intervals(self):
 
             for d, v in distance:
                 value = v.pop()
-                ret[d][0] = min(ret[d][0], value)
-                ret[d][1] = max(ret[d][1], value)
+                try:
+                    ret[d][0] = min(ret[d][0], value)
+                    ret[d][1] = max(ret[d][1], value)
+                except TypeError:
```

Comment:
Interesting, why can we now end up here with SymPy?

Reply:
I mean SymPy 1.12

Reply:
Somehow previous versions end up with `{0, expr}` as values but with sympy 1.12 you only get `{expr}`. Didn't look too deep into why though

Reply:
there might be a deeper issue then; are we now generating the same exact code for the MFE?

Reply:
We are we test all sympy version 1.9-1.12 that pass the same test